rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md

```diff
@@ -29,13 +27,12 @@ The code quality standard should extend the Visual Studio Analyser. A wide varie
 Search & Install the NuGet packages:
 
 * "Roslyn Security Guard" ([Nuget page for Roslyn Security Guard](https://www.nuget.org/packages/RoslynSecurityGuard/)) - Security audit on .NET Applications.
-* "StyleCop.Analysers" ([Nuget page for StyleCop.Analysers](https://www.nuget.org/packages/StyleCop.Analyzers/1.0.0)) - Ensures C# code style conformity.
-* "tslint" ([Nuget page for tslint](https://www.nuget.org/packages/tslint/)) - Allows for issue tracking of TypeScript projects.
+* (Optional) "StyleCop.Analysers" ([Nuget page for StyleCop.Analysers](https://www.nuget.org/packages/StyleCop.Analyzers/1.0.0)) - Ensures C# code style conformity. It is not compatible with Visual Studio 2022
```

Comment:
> It is not compatible with Visual Studio 2022

Should we still reccommend this?

Reply:
I think - no. Can I remove it?

Reply:
If it's not usable in a modern development setting I'd get rid of it/replace it with something worth mentioning

Reply:
Done - removed recommendation for StyleCop.Analysers. [In this rule](https://www.ssw.com.au/rules/consistent-code-style/) StyleCop  is bad example, so yeah, we shouldn't mention it anymore